XamarinCommunityToolkit/Views/CameraView/CameraView.shared.cs

```diff
@@ -1,18 +1,39 @@
 ﻿using System;
-using System.ComponentModel;
+using Xamarin.CommunityToolkit.Helpers;
 using Xamarin.Forms;
 
 namespace Xamarin.CommunityToolkit.UI.Views
 {
 	public class CameraView : View
 	{
-		public event EventHandler<MediaCapturedEventArgs> MediaCaptured;
+		readonly WeakEventManager<MediaCapturedEventArgs> mediaCapturedEventManager = new WeakEventManager<MediaCapturedEventArgs>();
```

Comment:
Can you do this change in other PR? I think this PR should be just an implementation of `AsyncCommand`

Reply:
No problem ✅